src/applications/personalization/profile/components/veteran-status-card/VeteranStatusAlerts.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CONTACTS } from '@department-of-veterans-affairs/component-library/contacts';
+
+const buildContactElements = item => {
+  const contactNumber = `${CONTACTS.DS_LOGON.slice(
+    0,
+    3,
+  )}-${CONTACTS.DS_LOGON.slice(3, 6)}-${CONTACTS.DS_LOGON.slice(6)}`;
+  const startIndex = item.indexOf(contactNumber);
+
+  if (startIndex === -1) {
+    return item;
+  }
+
+  const before = item.slice(0, startIndex);
+  const telephone = item.slice(
+    startIndex,
+    startIndex + contactNumber.length + 11,
+  );
+  const after = item.slice(startIndex + telephone.length);
+
+  return (
+    <>
+      {before}
+      <va-telephone contact={contactNumber} /> (
+      <va-telephone contact={CONTACTS[711]} tty />){after}
+    </>
+  );
+};
+
+const VeteranStatusAlert = ({ headline, messages, status = 'error' }) => {
+  return (
+    <va-alert
+      class="vads-u-margin-bottom--4"
+      close-btn-aria-label="Close notification"
+      status={status}
+      uswds
```

Comment:
Just an FYI: The `uswds` prop is no longer necessary for any component-library web components.